FunctionalTableDataDemo/Cells/UIControl+Extensions.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  UIControl+Extensions.swift
+//  Shopify
+//
+//  Created by Tom Burns on 2016-01-06.
```

Comment:
@boourns

Reply:
I figured you would have deleted all my code by now! :D